opensfm/reconstruction.py

```diff
@@ -317,7 +318,7 @@ def bundle_local(graph, reconstruction, gcp, central_shot_id, config):
         point = reconstruction.points[point]
         p = ba.get_point(point.id)
         point.coordinates = [p.p[0], p.p[1], p.p[2]]
-        point.reprojection_error = p.reprojection_error
+        point.reprojection_errors = p.reprojection_errors
```

Comment:
The Point type has a `reprojection_error` member that is stored and loaded from JSON files. The new `reprojection_errors` is not stored as of now.

It is convenient not to store the `reprojection_errors` dict since it is going to be large. In this case, we can remove the code to store the old `reprojection_error`.

Reply:
So we could either store the max of the dict of now ? And set to empty on load ?

Reply:
I think we can skip storing anything since we are not loading it